client/state/ui/selectors.js

```diff
@@ -78,3 +78,17 @@ export function isSectionIsomorphic( state ) {
 export function isPreviewShowing( state ) {
 	return get( state.ui, 'isPreviewShowing', false );
 }
+
+/**
+ * Returns a log of actions from certain types that have previously been
+ * dispatched for the current user.
+ *
+ * These actions are to be consumed by and inform Calypso's Guided Tours
+ * framework.
+ *
+ * @param  {Object}   state      Global state tree
+ * @return {Array}               Array of Redux actions, each with timestamp
+ */
+export function getActionLog( state ) {
+	return get( state.ui, 'actionLog', [] );
```

Comment:
>I agree, but I was going for consistency with the rest of the file. Any idea why most of the simpler selectors in there use `get`?

Personally, I use `get` only when I can't be confident that calling a property won't raise an "Uncaught TypeError: Cannot read property 'foo' of null/undefined" error. In the other selectors here, the shape of the state value can vary, e.g. `section` can be `false` or an object (which is not great, TBH).